helm/csi-vxflexos/Chart.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: v2
 appVersion: "2.1.0"
 kubeVersion: ">= 1.21.0 < 1.24.0"
+# If you are using a complex K8s version like "v1.21.3-mirantis-1", use this kubeVersion check instead
```

Comment:
Did you want to reword complex version to extended version, so they are consistent with the doc changes you are making? Just a suggestion, you can ignore this, not a must change imo.